.github/workflows/pipeline.yml

```diff
@@ -28,68 +28,67 @@ jobs:
-            --name ${{ secrets.SVC_CLUSTER_NAME }} \
-            --overwrite-existing
-
       # Install kubelogin (needed for AAD-enabled clusters)
       - name: Install kubelogin
         run: |
           az aks install-cli
           kubelogin --version
 
-      # Convert kubeconfig for non-interactive login (GitHub runner)
-      - name: Convert kubeconfig for kubelogin
-        run: |
-          kubelogin convert-kubeconfig -l azurecli
-
-      # Verify kubectl works
-      - name: Check cluster access
-        run: kubectl get nodes
-
-  mgmt-cluster-access:
-    runs-on: ubuntu-latest
-    permissions:
-      id-token: 'write'
-      contents: 'read'
-
-    steps:
-      - name: Checkout repo
-        uses: actions/checkout@v4
-
-      # Login to Azure using OIDC (no secret needed if federated identity is set up)
-      - name: Azure Login
-        uses: azure/login@v2
-        with:
-          client-id: ${{ secrets.AZURE_CLIENT_ID }}
-          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
-          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
-          # if not using federated identity:
-          # client-secret: ${{ secrets.AZURE_CLIENT_SECRET }}
-
       # Get AKS credentials
       - name: Get AKS credentials
         run: |
+          az aks get-credentials \
+            --resource-group ${{ secrets.SVC_RESOURCE_GROUP }} \
+            --name ${{ secrets.SVC_CLUSTER_NAME }} \
+            --overwrite-existing \
+            -f ./svc-cluster.kubeconfig
           az aks get-credentials \
             --resource-group ${{ secrets.MGMT_RESOURCE_GROUP }} \
             --name ${{ secrets.MGMT_CLUSTER_NAME }} \
-            --overwrite-existing
-
-      # Install kubelogin (needed for AAD-enabled clusters)
-      - name: Install kubelogin
-        run: |
-          az aks install-cli
-          kubelogin --version
+            --overwrite-existing \
+            -f ./mgmt-cluster.kubeconfig
 
       # Convert kubeconfig for non-interactive login (GitHub runner)
       - name: Convert kubeconfig for kubelogin
         run: |
-          kubelogin convert-kubeconfig -l azurecli
+          kubelogin convert-kubeconfig -l azurecli --kubeconfig ./svc-cluster.kubeconfig
+          kubelogin convert-kubeconfig -l azurecli --kubeconfig ./mgmt-cluster.kubeconfig
 
       # Verify kubectl works
       - name: Check cluster access
-        run: kubectl get nodes
+        run: |
+          kubectl --kubeconfig ./svc-cluster.kubeconfig get pods -A -l app=maestro
+          kubectl --kubeconfig ./svc-cluster.kubeconfig get service -n maestro
+          kubectl --kubeconfig ./mgmt-cluster.kubeconfig get pods -A -l app=maestro-agent
+ 
+      - name: patch the istio
+        run: |
+          kubectl --kubeconfig ./svc-cluster.kubeconfig patch deploy/maestro -n maestro -p '{"spec":{"template":{"metadata":{"annotations":{"traffic.sidecar.istio.io/excludeInboundPorts":"8090","traffic.sidecar.istio.io/excludeOutboundPorts":"8090"}}}}}'
```

Comment:
we should also exclude `8000` port, since istio enable extra auth on the port, that's maybe why the consumer tests fail.

Reply:
Thanks @morvencao , will add the 8000 and retry.